src/main/webapp/app/modules/provider/record/record-edit.tsx

```diff
@@ -447,14 +444,15 @@ export class RecordEdit extends React.Component<IRecordEditViewProp, IRecordEdit
       this.setState({ hiddenLocations: tail });
     }
     services.forEach(service => (service['locationIndexes'] = []));
+    const zeroLocations = locations.length === 0;
```

Comment:
I think it should be named `hasLocations` (and check if > 0) and then in ifs use `hasLocations ? what if yes : what if no`